Tasks/PublishBuildArtifacts/task.json

```diff
@@ -50,9 +50,9 @@
       "name": "TargetPath",
       "type": "string",
       "label": "File share path",
-      "defaultValue": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)",
-      "required": false,
-      "helpMarkDown": "The file share to which the artifact files will be copied. Publishing artifacts from a Linux or macOS agent to a file share is not supported.",
+      "defaultValue": "",
+      "required": true,
```

Comment:
Do we need to bump the major version for this 'required' change?

Reply:
In my opinion we don't need to bump the major version because the task would have failed before if this value was missing.  The UI just didn't indicate that it was required.

Reply:
I agree with David.  A new major version should be reserved for big investments, rewrites etc...

Reply:
Yup, no impact for existing users of the task, so I am not bumping up the major version.